### PR TITLE
never preview dos:// urls [risk: low]

### PR DESCRIPTION
**Pull request by davidangb**
**Expected**: dos:// urls never show contents in their file preview.

**Actual**: if the dos:// url resolves to an otherwise-previewable file, such as a .txt file, the preview modal shows the "Loading preview" spinner infinitely, and never makes an ajax call to get file contents. The url `dos://broad-dsp-dos.storage.googleapis.com/preview_dos.json` illustrates this in the dev env.

In discussion with PO, decision is - at least in the short term - to never show previews for DOS objects. We may re-enable those previews in the future, but we are turning them off entirely for now.

-----

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by gpolumbo-broad — APPROVED** [review]

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/previewmodal/PreviewSpec.scala [hunk 1]
```diff
@@ -124,7 +124,7 @@ class PreviewSpec extends FreeSpec with ParallelTestExecution with WebBrowserSpe
     }
   }
 
-  "Preview Modal should display correct message and preview when file is accessible for dos:// link" ignore {
+  "Preview Modal should hide preview even when dos:// link resolves to an otherwise previewable file" in {
```
Comment (gpolumbo-broad) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/previewmodal/PreviewSpec.scala:127]: Does this read clearer:  "Preview Modal should hide preview for all dos:// links"

**Review by gpolumbo-broad — APPROVED** [review]

**Comment by zarsky-broad** [comment]
I'd like to hold off on merging this until we can verify that the initial issue wasn't just caused by bad metadata. I've changed the file in question and am waiting for it to propagate through the system now.

**Review by zarsky-broad — APPROVED** [review]
We figured out why we didn't mean to support this, pet service accounts won't necessarily have access to the buckets where the original files are stored, so we can't preview like this.

**Comment by davidangb** [comment]
Most recent test run had one failure:
```
[info] *** 1 TEST FAILED ***
[error] Failed: Total 123, Failed 1, Errors 0, Passed 122, Ignored 7
[error] Failed tests:
[error] 	org.broadinstitute.dsde.firecloud.test.security.AuthDomainMultiGroupsSpec
```
which was a `412 Precondition Failed` failure coming out of Sam.

Between the most recent test run and the run previous to that, all tests passed, so I'm calling it complete with no need to re-run the failed test.

**Pull request merged**